[IAM-5169] add docs for the migration process to the new SSO (#1940)
saucelabs/sauce-docs: docs/basics/acct-team-mgmt/org-settings.md

:::caution
This SSO flow has been `deprecated`. Use this documentation only if your organization was not migrated to the new SSO.

> If you do not currently have any SSO integrations set up at Sauce Labs and you wish to establish a new one, you can refer to our [new documentation](/basics/sso/setting-up-sso) on setting up SSO.

Reviewer: ```suggestion
If you do not have any SSO integrations set up at Sauce Labs and you wish to establish a new integration, see [Setting Up SSO](/basics/sso/setting-up-sso) to get started.
```

Author: fixed

> If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, please follow our [step-by-step migration tutorial](/basics/sso/migration-from-deprecated-sso/) for a smooth transition.

Reviewer: ```suggestion
If you have previously implemented the deprecated SSO integration and wish to migrate to the new SSO implementation, see the [step-by-step migration guide](/basics/sso/migration-from-deprecated-sso/).
```

Author: fixed
